src/theme/components/header/headerSearchForm.ts

```diff
@@ -26,16 +21,21 @@ const HeaderSearchForm = {
       maxWidth: { md: "1312px" },
       whiteSpace: "initial",
       "> div": {
+        margin: "20px 15px 0",
         marginLeft: { md: "m", lg: "140px" },
         marginRight: { md: "m" },
       },
     },
-    fieldset: {
-      margin: { base: "20px 15px 0", md: "0" },
+    "#radio-group-search-type": {
+      margin: { base: "0 20px 20px", md: "0" },
     },
     textInput: {
       label: {
-        fontSize: { md: "18px" },
+        fontSize: {
+          base: "heading.callout",
+          sm: "heading.tertiary",
```

Comment:
This didn't work out as well as I hoped.  Instead, let's drop the `sm` value altogether and just bounce directly from `callout` to `secondary`.
```
fontSize: { base: "heading.callout", md: "heading.secondary" },
```